fhirstore/fhirstore.py

```diff
@@ -314,33 +319,45 @@ def search(
                 {"resource": h["_source"], "search": {"mode": "match"}}
                 for h in hits["hits"]["hits"]
             ],
-
             "total": hits["hits"]["total"]["value"],
         }
 
         if elements:
             bundle["tag"] = {"code": "SUBSETTED"}
 
         if include:
-            #For each result instance
+            included_hits = {}
+            # For each result instance
             for item in bundle["items"]:
-                #For each attribute to include
+                # For each attribute to include
                 for attribute in include:
-                    # split the reference attribute "Practioner/123" into a 
+                    # split the reference attribute "Practioner/123" into a
                     # resource "Practioner" and an id "123"
-                    included_resource, included_id = re.split(
-                        "\/", item[attribute]["reference"], maxsplit=1
-                    )
+                    try:
+                        included_resource, included_id = re.split(
```

Comment:
nit: I think it's better to use Python split instead of re.split if you can

Reply:
Noted :)